app/jobs/europeana/feeds/fetch_job.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Europeana
+  module Feeds
+    class FetchJob < ActiveJob::Base
+      queue_as :feeds
+
+      def perform(url, download_media = false)
+        @url = url
+        @download_media = download_media
```

Comment:
This does nothing inside the engine/gem itself and should be removed, along with the argument.